packages/@react-spectrum/dialog/docs/DialogTrigger.mdx

```diff
@@ -0,0 +1,464 @@
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['DialogTrigger']} />
+
+## Example
+
+```tsx example
+<DialogTrigger type="popover">
+  <ActionButton>Disk Status</ActionButton>
+  <Dialog>
+    <Heading>C://</Heading>
+    <Divider />
+    <Content>
+      <Text>
+        50% disk space remaining.
+      </Text>
+    </Content>
+  </Dialog>
+</DialogTrigger>
+```
+
+## Content
+
+The DialogTrigger accepts exactly two children: the element which triggers the opening
+of the Dialog and the Dialog itself. The trigger must be the first child 
+passed into the DialogTrigger and should be an element that supports press events.
+
+If your Dialog has buttons within it that should close the Dialog when pressed,
+you must wrap the Dialog in a function in order to properly propagate the DialogTrigger's
+`close` state setter to the Dialog's children. Dialogs that do not contain such interactive
+elements can simply provide the Dialog component as is to the DialogTrigger as its second child.  
+
+The example below demonstrates how to pass the DialogTrigger's `close` state setter to the Dialog's buttons. 
+
+```tsx example
+<DialogTrigger>
+  <ActionButton>Checkout</ActionButton>
+  {(close) => ( 
+    <Dialog>
+      <Heading>Confirm checkout?</Heading>
+      <Divider />
+      <Content>
+        <Text>
+          You have 5 items in your cart. Proceed to checkout?
+        </Text>
+      </Content>
+      <ButtonGroup>
+        <Button variant="secondary" onPress={close}>Cancel</Button>
+        <Button variant="cta" onPress={close} autoFocus>Confirm</Button>
+      </ButtonGroup>
+    </Dialog>
+  )}
+</DialogTrigger>
+```
+
+## Dialog Types
+
+By providing a `type` prop, you can specify the type of Dialog that is rendered
+by your DialogTrigger. Note that pressing the `esc` key will close the Dialog regardless
+of its `type`.
+
+### Modal
+
+Modal Dialogs create an underlay that blocks access to the underlying user interface
+until the Dialog is closed. Sizing options can be found in the [Dialog page](./Dialog.html#size).
+Focus is trapped inside the Modal as per the [accessibility guidelines](https://www.w3.org/TR/wai-aria-practices/#dialog_modal)
+laid out by W3C.
+
+```tsx example
+<DialogTrigger type="modal">
+  <ActionButton>Unlink</ActionButton>
+  {(close) => ( 
+    <Dialog>
+      <Heading>Unlinking email</Heading>
+      <Divider />
+      <Content>
+        <Text>
+          This will unlink your email from your profile "TestUser". Are you sure?
+        </Text>
+      </Content>
+      <ButtonGroup>
+        <Button variant="secondary" onPress={close}>Cancel</Button>
+        <Button variant="cta" onPress={close} autoFocus>Confirm</Button>
+      </ButtonGroup>    
+    </Dialog>
+  )}
+</DialogTrigger>
+```
+
+### Popover
+
+If you want a Dialog that doesn't block access to the user interface, consider
+using a Popover Dialog. See the [docs below](#dialog-placement) for how you can
```

Comment:
I think this is the first time in the doc we capitalize a type, why is the type, 'popover' capitalized?

Reply:
I capitalized Popover, Modal, and Tray because they are actual components as well. Felt easier to read as well